### PR TITLE
SONARJAVA-1030 Add support for fields

### PR DESCRIPTION
**Pull request by benzonico**

### CODE REVIEW

File: java-checks/src/main/java/org/sonar/java/checks/LocksNotUnlockedCheck.java [hunk 1]
```diff
@@ -66,5 +68,23 @@ public void visitNode(Tree tree) {
   }
 
 
+  private List<Symbol> getAccessibleLockFields(Symbol.MethodSymbol symbol) {
+    List<Symbol> symbols = Lists.newArrayList();
+    Symbol owner =  symbol.owner();
+    while (owner.isTypeSymbol()) {
+      Symbol.TypeSymbol typeSymbol = ((Symbol.TypeSymbol) owner);
```
Comment (SonarTech) [inline java-checks/src/main/java/org/sonar/java/checks/LocksNotUnlockedCheck.java]: :new: MAJORRemove those useless parentheses.[](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessParenthesesCheck)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 new issue:
* 1 major

Details:
* [Remove those useless parentheses.](https://github.com/SonarSource/sonar-java/blob/b86295c5aeb46ceab3fccfbf735f5ff02aa0a329/java-checks/src/main/java/org/sonar/java/checks/LocksNotUnlockedCheck.java#L75)[](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessParenthesesCheck)

**Comment by SonarTech** [comment]
SonarQube analysis reported no new issues.

**Comment by m-g-sonar** [comment]
LGTM!

**Pull request closed without merging**